src/sparsezoo/analyze_v2/model_validator/parameter_analysis_model.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+from pydantic import BaseModel, Field
+
+from sparsezoo.analyze_v2.model_validator.distribution_analysis_model import (
+    DistributionAnalysisModel,
+)
+from sparsezoo.analyze_v2.model_validator.quantization_analysis_model import (
+    QuantizationAnalysisModel,
+)
+from sparsezoo.analyze_v2.model_validator.sparsity_analysis_model import (
+    SparsityAnalysisModel,
+)
+
+
+class ParameterAnalysisModel(BaseModel):
```

Comment:
we don't need `Model` in all the subclass names since the rest tells us what it really is and the base pydantic class is more there for convienence (ie we wouldn't name a dataclass as `_DataClass`)